dashboard/utils_test.py

```diff
@@ -733,6 +735,21 @@ def test_get_best_final_grade_for_course(self):
             FinalGradeFactory.create(user=self.user, course_run=course_run, grade=grade, passed=True)
         assert mmtrack.get_best_final_grade_for_course(finaid_course).grade == 0.8
 
+    @override_settings(FEATURES={"USE_COMBINED_FINAL_GRADE": True})
+    def test_get_overall_final_grade_for_course(self):
+        """
+        Test for get_overall_final_grade_for_course to return CombinedFinalGrade for course
+        """
+        mmtrack = MMTrack(
+            user=self.user,
+            program=self.program_financial_aid,
+            edx_user_data=self.cached_edx_user_data
+        )
+        finaid_course = self.crun_fa.course
+        assert mmtrack.get_overall_final_grade_for_course(finaid_course) == ""
```

Comment:
Might be good to parametrize this test (or add more tests) to check for result when: 
  - `Course.has_exam` is True or False
  - `MMTrack.get_best_proctored_exam_grade` is None or between 0-1
  -  Grades for course runs exist or not

Reply:
That part that is under `else` we were going to get rid of once CombinedFinalGrade is populated in production. I can remove it in this PR if you prefer

Reply:
OK, never mind then.